app/views/categories/_form.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_with do |form| %>
+<% debugger %>
```

Comment:
why are we NOT reviewing our PRs??
Debugger in a PR is absolutely not acceptable

Reply:
done